.devcontainer/Dockerfile

```
@@ -3,7 +3,7 @@
 FROM rocker/tidyverse:4.4.1
 
 # Install epiworldR
-RUN install2.r epiworldR
+RUN installGithub.r UofUEpiBio/epiworldR
```

Your container is not re-building with the right version b/c it is caching the line. We can point to the specific commit we want to ensure it updates (and doesn't do caching):

```suggestion
RUN installGithub.r UofUEpiBio/epiworldR@cc673d1
```

---

Thanks, this fixed the issue.